flows/all_retirements.py

```diff
@@ -9,30 +9,40 @@
 
 @task()
 def fetch_all_retirements_task():
-    """Builds Verra data"""
+    """Builds Merged Retirements data"""
     verra_df = utils.get_latest_dataframe("verra_retirements")
     klima_df = utils.get_latest_dataframe("polygon_klima_retirements")
+
     # Verra manipulation
     verra_df = verra_df[['retirement_beneficiary',
                          'retirement_date',
                          'serial_number',
+                         'bridge',
+                         'project_id',
                          'quantity']].copy()
-    verra_df["quantity"] = verra_df["quantity"] / 1000
     verra_df = verra_df.rename(
         columns={
-            'retirement_beneficiary': 'beneficiary',
-            'serial_number': 'proof'
```

Comment:
is serial number no longer being included at all? I'm confused about why we're removing this logic and setting the column to None in two places?